docs/UserGuide.md

```diff
@@ -36,10 +36,10 @@ Please read through sections [Installation](#installation) and [Command Breakdow
  
    <img src="images/Ui.png" width="650">
 
-5. Type the command in the command box and press `Enter` to execute it. e.g. typing **`help`** and pressing `Enter` will
-   open the help window.
+5. Type a command in the command box and press `Enter` to execute it. e.g. typing `help` and pressing `Enter` will
+   execute the `help` command and open the help window.
 
-   To get started with UniCa$h, you can run the `add_transactions` command!
+   To get started with UniCa$h, you can run the (`add_transactions` command)[#add-transaction]!
```

Comment:
<img width="927" alt="Screenshot 2023-11-13 at 16 42 53" src="https://github.com/AY2324S1-CS2103-T16-3/tp/assets/29497029/16f2b668-93f9-4453-9e56-41f7ba3c1269">

This link is not working, believe you flipped the order of parentheses: i.e. `[]()`

Reply:
🤦‍♂️